sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -1928,6 +1928,15 @@ class SubquerySuite extends QueryTest
     checkAnswer(df, Nil)
   }
 
+  test("SPARK-43156: scalar subquery with Literal result like `COUNT(1) is null`") {
```

Comment:
this is no longer needed as we have golden file test

Reply:
Removed